src/common/options/rgw.yaml.in

```diff
@@ -796,6 +796,14 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: rgw_jaeger_enable
+  type: bool
+  level: advanced
+  desc: should RGW uses jaeger tracing system 
+  default: true
+  services:
+  - rgw
+  with_legacy: true
```

Comment:
what does `with_legacy` mean?

Reply:
when true, this generates a legacy option that can be accessed via `g_conf->rgw_jaeger_enable`

Reply:
thanks. so, since we don't use this access mode, we should set `with_legacy` to "false"